lib/cocoapods-core/specification/consumer.rb

```diff
@@ -362,6 +367,30 @@ def _prepare_prefix_header_contents(value)
         end
       end
 
+      # Normalizes script phases value into an array of hashes where each hash represents the entire script phase.
+      #
+      # @param  [Hash{String=>String}] value.
+      #         The value of the attribute as specified by the user.
+      #
+      # @return [Array<Hash{String=>String}>] the normalized script phases array.
+      #
+      def _prepare_script_phases(value)
+        if value
```

Comment:
`return unless value`

Reply:
was cargo culting from the remaining methods :)